lib/internal/test_runner/test.js

```diff
@@ -998,7 +998,6 @@ class Test extends AsyncResource {
       process.exit();
     }
   }
-
```

Comment:
Unrelated

Reply:
Also whoops. Happened when removing the old `getFiles()`